app/models/ticket.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Ticket < ApplicationRecord
+  enum :ticket_type, {
+    edit_person: "edit_person",
+  }
```

Comment:
Does this column provide any actional value, on top of `metadata_type`?
Or in other words, can you think of use-cases where your suggested `ticket_type` is significantly different from `metadata_type` (defined by the polymorphic `metadata` association below)?

Reply:
Hm, I see that enums are useful for switch-case, but this doesn't change the fact that the contained information is basically redundant.

In the places where you're currently doing the switch-case, can you instead maybe try to do something like `if ticket.metadata.is_a? EditPersonTicket` or similar?

Reply:
Yeah, that's basically an enum without enforcing it in Rails.
What about moving it into a constant like
```TICKET_TYPES = {
  edit_person: "EditPersonTicket",
  anonymization: "AnonymizationTicket",
}.freeze
```

Reply:
That's better. Changed, thank you.